dockers/docker-orchagent/base_image_files/cfgmgr

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker exec -i swss cfgmgr "$@"
```

Comment:
it is better to incorporate this into the show/config commands in sonic utilities. I suggest to remove this in this PR and only include vlanmgrd and intfmgrd.

Reply:
removed.